modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -269,6 +269,43 @@ Card Shadow:: Specifies the shadow beneath the card.
 Card Width:: Specifies the card width.
 Card Height:: Specifies the card height.
 
+== ACM Search Box
+
+The ACM Search Box component enables users to search data integrated from both Community Cloud and Anypoint Platform.
```

Comment:
... enables users to search for content that exists in both Community Cloud, like articles, announcements and forum posts, and the Anypoint Platform, like APIs and client applications. Results from the search are presented in the ACM Search Results component [ADD LINK]

Reply:
Made this change. Much better! Thank you.